ui/src/features/VersionCanvas/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Map as YMap } from "yjs";
+
+import Canvas from "@flow/features/Canvas";
+import type { YWorkflow } from "@flow/lib/yjs/types";
+
+import { WorkflowTabs } from "../Editor/components";
+
+import useHooks from "./hooks";
+
+type Props = {
+  yWorkflows: YMap<YWorkflow>;
+};
+
+const VersionCanvas: React.FC<Props> = ({ yWorkflows }) => {
+  const {
+    currentWorkflowId,
+    isSubworkflow,
+    nodes,
+    edges,
+    openWorkflows,
+    handleWorkflowClose,
+    handleCurrentWorkflowIdChange,
+  } = useHooks({ yWorkflows });
+
+  return (
+    <div className="relative flex size-full flex-col">
+      <Canvas
+        readonly
+        isSubworkflow={isSubworkflow}
+        nodes={nodes}
+        edges={edges}
+      />
+      <WorkflowTabs
+        openWorkflows={openWorkflows}
+        currentWorkflowId={currentWorkflowId}
+        onWorkflowClose={handleWorkflowClose}
+        onWorkflowChange={handleCurrentWorkflowIdChange}
+      />
+    </div>
```

Comment:
Here needs to be fixed up. I can never see the workflow tabs in the version preview, and the canvas grid doesn't seem to be anchored to anything. Check the regular Editor's use of canvas + the topbar to see how to fix this